modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -21,6 +21,11 @@ import * as Predicate from './util/Predicate';
  * @class tinymce.DragDropOverrides
  */
 
+// Arbitrary values needed when scrolling CEF elements
+const ScrollPixelsPerInterval = 32;
```

Comment:
These variables are also PascalCase when they should be camelCase. So please fix that when you add the other missing parts.